tests/cypress/e2e/features/masks_basics.js

```diff
@@ -167,4 +172,125 @@ context('Manipulations with masks', { scrollBehavior: false }, () => {
             cy.finishMaskDrawing();
         });
     });
+
+    describe('Empty masks actions', () => {
+        beforeEach(() => {
+            cy.removeAnnotations();
+        });
+
+        after(() => {
+            cy.removeAnnotations();
+        });
+
+        function checkEraseTools(baseTool = '.cvat-brush-tools-brush', disabled = true) {
+            cy.get(baseTool).should('have.class', 'cvat-brush-tools-active-tool');
+
+            const condition = disabled ? 'be.disabled' : 'not.be.disabled';
+            cy.get('.cvat-brush-tools-eraser').should(condition);
+            cy.get('.cvat-brush-tools-polygon-minus').should(condition);
+        }
+
+        it(
+            'Drawing a mask, fully erase it. Erase tools are blocked with empty mask. Empty shape is not created.',
+            () => {
```

Comment:
```suggestion
        it('Erase tools are locked when nothing to erase', () => {
```

Reply:
And additionally need to check, that erase tools a locked in the beginning of drawing.

Reply:
Added such check.